tests/integration/components/select-list-test.js

```diff
@@ -183,3 +183,15 @@ test('binds properly to `title` attribute', function(assert) {
 
   assert.equal(this.$('select').attr('title'), title);
 });
+
+test('remains invalid when prompt is selected', function(assert) {
+  var options = [];
+  this.set('options', options);
+  this.set('prompt', 'First prompt');
+
+  this.render(hbs`
+    {{select-list content=options value=selection prompt=prompt}}
+  `);
+
+  assert.equal(this.$('select option:selected')[0].validity.valid, false);
```

Comment:
True, but we don't have that api in other browsers, so the closest thing to do in order to test it is to set the `required` attribute and then check if the select element has a value. Perhaps testing for $('select').val() being truthy is a bit closer, but we can't make a validity test that would pass in either chrome or phantom js (which is what our two test runners are).